<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tree</class>
 <widget class="QWidget" name="Tree">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="boxActions"/>
   </item>
   <item>
    <widget class="QListWidget" name="pathlist">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DropOnly</enum>
     </property>
     <property name="flow">
      <enum>QListView::LeftToRight</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="movement">
      <enum>QListView::Snap</enum>
     </property>
     <property name="layoutMode">
      <enum>QListView::Batched</enum>
     </property>
     <property name="gridSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="editSearch"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnSearch">
       <property name="text">
        <string>Search</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/actions/search_small.png</normaloff>icons/actions/search_small.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="boxUpload"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
